SU2_CFD/src/solvers/CNEMONSSolver.cpp

```diff
@@ -1077,6 +1078,102 @@ void CNEMONSSolver::BC_Smoluchowski_Maxwell(CGeometry *geometry,
+                                                 unsigned short val_marker) {
+
+  /*--- Local variables ---*/
+  const auto Marker_Tag = config->GetMarker_All_TagBound(val_marker);
+  const bool implicit = (config->GetKind_TimeIntScheme() == EULER_IMPLICIT);
+  const su2double emissivity = config->Get_Emissivity(Marker_Tag);
+    
+  /*--- Get the locations of the primitive variables ---*/
+  const unsigned short T_INDEX       = nodes->GetTIndex();
+  const unsigned short TVE_INDEX     = nodes->GetTveIndex();
+  const unsigned short RHO_INDEX     = nodes->GetRhoIndex();
+  const unsigned short RHOCVTR_INDEX = nodes->GetRhoCvtrIndex();
+
+  /*--- Loop over all of the vertices on this boundary marker ---*/
+  SU2_OMP_FOR_DYN(OMP_MIN_SIZE)
+  for(auto iVertex = 0u; iVertex < geometry->nVertex[val_marker]; iVertex++) {
+
+    const auto iPoint = geometry->vertex[val_marker][iVertex]->GetNode();
+
+    /*--- Check if the node belongs to the domain (i.e, not a halo node) ---*/
+    if (!geometry->nodes->GetDomain(iPoint)) continue;
+
+    /*--- Compute dual-grid area and boundary normal ---*/
+    const auto Normal = geometry->vertex[val_marker][iVertex]->GetNormal();
+    su2double Area = GeometryToolbox::Norm(nDim, Normal);
+
+    /*--- Initialize the convective & viscous residuals to zero ---*/
+    for (auto iVar = 0u; iVar < nVar; iVar++) {Res_Visc[iVar] = 0.0;}
+
+    /*--- Set the residual on the boundary with the specified heat flux ---*/
+    // TODO: Look into this!
+    // Note: Contributions from qtr and qve are used for proportional control
+    //       to drive the solution toward the specified heatflux more quickly.
+    const auto V      = nodes->GetPrimitive(iPoint);
+    const auto GradV  = nodes->GetGradient_Primitive(iPoint);
+    su2double dTdn   = 0.0;
+    su2double dTvedn = 0.0;
+    for (auto iDim = 0u; iDim < nDim; iDim++) {
+      dTdn   += GradV[T_INDEX][iDim]*Normal[iDim];
+      dTvedn += GradV[TVE_INDEX][iDim]*Normal[iDim];
+    }
+    su2double ktr = nodes->GetThermalConductivity(iPoint);
+    su2double kve = nodes->GetThermalConductivity_ve(iPoint);
+
+    /*--- Scale thermal conductivity with turb ---*/
+    // TODO: Need to determine proper way to incorporate eddy viscosity
+    // This is only scaling Kve by same factor as ktr
+    su2double Mass = 0.0;
+    auto&     Ms   = FluidModel->GetSpeciesMolarMass();
+    su2double tmp1, scl, Cptr;
+    su2double Ru=1000.0*UNIVERSAL_GAS_CONSTANT;
+    su2double eddy_viscosity = nodes->GetEddyViscosity(iPoint);
+    for (unsigned short iSpecies=0; iSpecies<nSpecies; iSpecies++)
+      Mass += V[iSpecies]/V[RHO_INDEX]*Ms[iSpecies];
+    Cptr = V[RHOCVTR_INDEX]/V[RHO_INDEX]+Ru/Mass;
+    tmp1 = Cptr*(eddy_viscosity/Prandtl_Turb);
+    scl  = tmp1/ktr;
+    ktr += Cptr*(eddy_viscosity/Prandtl_Turb);
+    kve  = kve*(1.0+scl);
+    //Cpve = V[RHOCVVE_INDEX]+Ru/Mass;
+    //kve += Cpve*(val_eddy_viscosity/Prandtl_Turb);
```

Comment:
Im pretty sure this can be removed from all case BC's.   Eddy viscosity in NEMO is always zero at the wall.

Reply:
So remove lines 1130 - 1146?

Reply:
Yes exactly.